backend/api/Helpers/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Pims.Api.Helpers.Extensions
+{
+    public static class DateTimeExtensions
+    {
+        public static int ToFiscalYear(this DateTime dateTime)
```

Comment:
Where is the output required for this?  Currently where fiscal is relevant we only capture the year not the date.  It's also not ideal as it's hard-coded (although I wouldn't change it).

Reply:
I actually moved this logic from the backend to the frontend so this is unused. The idea is that I need to know the current fiscal when submitting the estimated/netbook from the frontend since the table only includes the value not the fiscalyear.